js/Genoverse.js

```diff
@@ -336,7 +339,7 @@ var Genoverse = Base.extend({
     var conf, j;
 
     for (var i = 0; i < this.tracks.length; i++) {
-      if (this.tracks[i].id && !(this.tracks[i] instanceof Genoverse.Track.Legend) && !(this.tracks[i] instanceof Genoverse.Track.HighlightRegion)) {
```

Comment:
This should check `HighlightRegion` as well, right? Looks like it's checking Legend both times here